packages/browser-agent-core/features/ajax/aggregate/index.js

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { registerHandler as register } from '../../../common/event-emitter/register-handler'
+import { stringify } from '../../../common/util/stringify'
+import { nullable, numeric, getAddStringContext, addCustomAttributes } from '../../../common/serialize/bel-serializer'
+import { handle } from '../../../common/event-emitter/handle'
+import { getConfigurationValue, getInfo } from '../../../common/config/config'
+import { HarvestScheduler } from '../../../common/harvest/harvest-scheduler'
+import { subscribeToUnload } from '../../../common/unload/unload'
+import { setDenyList, shouldCollectEvent } from '../../../common/deny-list/deny-list'
+import { FeatureBase } from '../../../common/util/feature-base'
+
+export class Aggregate extends FeatureBase {
+  constructor(agentIdentifier, aggregator, externalFeatures = []) {
+    super(agentIdentifier, aggregator, externalFeatures)
+    let ajaxEvents = []
+    let spaAjaxEvents = {}
+    let sentAjaxEvents = []
+    let scheduler
+
```

Comment:
Missing runtime check for whether instrumentation was loaded:

https://github.com/newrelic/newrelic-browser-agent/blob/e0addb7c19e52429cc17b50a4e6f01892b3c1fc1/feature/xhr/aggregate/index.js#L26-L31

Reply:
The flag was needed before the modularity work, because features initialized as a side-effect of import.  Now we can be more direct when initializing the class instances.